src/main/java/ch/njol/skript/SkriptCommand.java

```diff
@@ -90,30 +92,45 @@ public class SkriptCommand implements CommandExecutor {
 		if (TestMode.DEV_MODE)
 			SKRIPT_COMMAND_HELP.add("test");
 	}
-	
-	private static void reloading(CommandSender sender, String what, Object... args) {
+
+	private static void reloading(CommandSender sender, String what, RedirectingLogHandler logHandler, Object... args) {
+		String prefix = args.length == 0 ? Language.get(CONFIG_NODE + ".reload." + "player reload") : Language.format(CONFIG_NODE + ".reload." + "player reload", sender.getName());
 		what = args.length == 0 ? Language.get(CONFIG_NODE + ".reload." + what) : Language.format(CONFIG_NODE + ".reload." + what, args);
-		Skript.info(sender, StringUtils.fixCapitalization(m_reloading.toString(what)));
+		String message = StringUtils.fixCapitalization(m_reloading.toString(what));
+		Skript.info(sender, message);
+		if (SkriptConfig.sendReloadingInfoToOps.value()) {
+			String lowerCaseMessage = lowerCaseFirstChar(message);
+			notifyOperators(prefix + " " + lowerCaseMessage, sender, logHandler);
```

Comment:
I don't think it is a good idea to combine two language entries together like this. While this works for English, there are no guarantees for other languages. It'd be better to have the lang entry represent the entire message.